src/detect-engine-threshold.c

```diff
@@ -223,12 +223,17 @@ uint32_t ThresholdsExpire(const SCTime_t ts)
 
 #include "util-hash.h"
 
+#define TC_ADDRESS 0
```

Comment:
Is it the same as #define TRACK ?

Reply:
not understanding the question

Reply:
There are 2 series of 5 defines that look very much alike except maybe one element. Could they be merged together ?